.github/workflows/publish.yml

```diff
@@ -35,9 +35,15 @@ jobs:
         env:
           GCS_CREDENTIALS_KEY: ${{ secrets.GCS_CREDENTIALS_KEY }}
 
+      - name: Decrypt GCAR credentials
+        run: ./config/scripts/decrypt.sh "$MAVEN_PUBLISHER_KEY" ./maven-publisher.json.gpg ./maven-publisher.json
+        env:
+          MAVEN_PUBLISHER_KEY: ${{ secrets.MAVEN_PUBLISHER_KEY }}
+
       - name: Publish artifacts to Maven
-        run: ./gradlew build publish --stacktrace
+        run: ./gradlew assemble publish --stacktrace
```

Comment:
@dmdashenkov, what's the difference with the previously made `build` call?

Reply:
`asseble` is faster, as it does not involve running tests. We assume that tests have already passed when we merge the branch into `mater` and publish the artifacts.